_posts/2019-09-03-release-1.8.2.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Apache Flink 1.8.2 Released"
+date:   2019-09-03 12:00:00
```

Comment:
The date should be update when the VOTE finish.

Reply:
Yes.